content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -207,7 +207,7 @@ kubectl rollout history deployment/frontend                      # Check the his
 kubectl rollout undo deployment/frontend                         # Rollback to the previous deployment
 kubectl rollout undo deployment/frontend --to-revision=2         # Rollback to a specific revision
 kubectl rollout status -w deployment/frontend                    # Watch rolling update status of "frontend" deployment until completion
-
+kubectl rollout restart deployment/frontend                      # Roll-out restart the "frontend" deployment
```

Comment:
Optionally, how about adding something like this?
```markdown
kubectl rollout restart deployment/frontend                      # Roll-out restart the "frontend" deployment
                                                                 # This will restart each Pod in turn
```

(fine to merge as-is, in my view; @apelisse I'm wondering if you think this is clearer)

Also: if I'm _wrong_ about what happens, eg it's actually Pod replacement happening, then I think that makes a stronger case for adding a note about the actual behavior.

Reply:
That's indeed what happens (mostly, we can't say if it's one-by-one since it depends on other parameters). But it'd be much more verbose than any other lines in this section. The word "Roll-out" already infers this. Thanks for the suggestion though.

Reply:
In wider IT jargon that's called a _rolling_ restart. It's just Kubernetes that calls this a rollout restart.

Reply:
I'm fine with that, are good on ` # Rolling restart the "frontend" deployment`?

Reply:
@apelisse that'd sort it, in just one line. IMO.